### ISSUE TITLE
App crashes

### ISSUE THREAD

**Issue by swetshaw** [amitshekhariitbhu/AndroidTensorFlowMachineLearningExample#7]
The App crashes when I am trying to run it using my own graph file and labels file.
I tweaked the code in ***mainActivity.java*** accordingly.

Is the app hardcoded to run using only your graph and label files?

**Comment by amitshekhariitbhu** [comment]
@swetshaw yes, you will have to change the input and output in the class `TensorFlowImageClassifier`.

**Comment by vaibhav-mehta1001** [comment]
I am getting the following error :

**Comment by vaibhavmehta1** [comment]
I am getting the following error when I downloaded and opened your app : 
Gradle sync failed: Cause: error in opening zip file
Consult IDE log for more details (Help | Show Log)

Is this is a problem with gradle or is it because I did not follow the steps on your blog post, and directly 
downloaded this ?
